### DISCUSSION TITLE
How do you use content translations?

### DISCUSSION THREAD

**Discussion by navarrofracisco** [directus/directus#7338] [Archive]
Did I miss something in the documentation? https://docs.directus.io/concepts/translations/#content-translations
Using translations is not intuitive at all. Here are my steps:

1. I created a simple articles collections.
2. I added a translations "field" to this articles. These created two collections: languages and articles_relations (hidden).
3. When I create a new article how do I enter a translated title? I see the list of Article Translations at the bottom, but does not do anything.

Thank you,
<img width="1008" alt="test-2" src="https://user-images.githubusercontent.com/8353831/128949900-8613430b-702f-484a-b0d7-84cb5e9e9b1e.PNG">
<img width="1003" alt="test-1" src="https://user-images.githubusercontent.com/8353831/128949902-47195c4e-37c1-40b5-98f9-514e73c56269.PNG">

**Comment by benhaynes** [comment]
Clicking on items in the article translations repeater should open them in a drawer for editing the translations. Do you have fields set up on the translation collection? Does anything happen, either in the App or dev-tools, when you click those items?

**Reply by navarrofracisco** [reply]
Thank you for your response, I was able to set them up.

**Comment by DLowHP** [comment]
I have a similar problem. 
I created data simple data model and added translations field. It displays a list of languages but if i click, for example, `de-DE` it just displays empty drawer.
![image](https://user-images.githubusercontent.com/65829376/129352386-4ccb7d2b-a04c-44f0-88cf-0ac0500dfeed.png)
![image](https://user-images.githubusercontent.com/65829376/129352420-8cd8c8ea-cac3-4a26-9478-870c909d411a.png)

**Reply by DLowHP** [reply]
Also, when loading collection item it makes a request to [/items/test_translations](http://localhost/directus/items/test_translations?fields[]=id&fields[]=languages_code&filter=%7B%22id%22:%7B%22_in%22:[]%7D%7D) which ends with [400] Bad request, and an error message:
```
{
    "errors": [
        {
            "message": "\"_in\" has to be an array of values",
            "extensions": {
                "code": "INVALID_QUERY"
            }
        }
    ]
}
```

**Reply by rijkvanzanten** [reply]
@DLowHP Do you have any fields in your translations collection? (`test_translations`)

**Reply by DLowHP** [reply]
Yes, it was generated by Directus
![image](https://user-images.githubusercontent.com/65829376/129366915-137e4496-6a84-4fce-b27c-823f0f9f1996.png)

**Reply by rijkvanzanten** [reply]
Yeah, and all hidden 🙂 

Try adding a new field, like `title`, in here. Those are the fields that then have a custom value per translation 👍🏻

**Reply by DLowHP** [reply]
Oh okey, so when I want to created multiple translatable fields I need to add them to `{collection}_translations` instead of `{collection}` data model?

**Reply by rijkvanzanten** [reply]
Correct! Translations relies on the following (example) structure:

```yaml
// articles

id: 1
internal_note: This is shared for every language
translations: // articles_translations
  - language: 'en-US'
    title: 'Good morning!'
  - language: 'nl-NL'
    title: 'Goedemorgen!'
```

**Reply by DLowHP** [reply]
I understand now, thank you very much and have a nice day :)

**Reply by rijkvanzanten** [reply]
You too!

**Comment by navarrofracisco** [comment]
Thank you for the responses. I was able to set up the translations. Not to self promote, but I wrote a step by step blog: https://blog.lunarbyte.io/2021/08/11/directus-content-translations/

I think the translations documentations could be improved, I will try to contribute to the docs when I can. Thank you!

**Reply by benhaynes** [reply]
That's not self-promotion, that's just a great article. Thank you for sharing! Feel free to reach out to me on Discord (https://directus.chat) if you'd like to coordinate more content creation! ❤️
